Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -552,8 +552,12 @@ void doEsirkepovDepositionShapeN (const GetParticlePosition& GetPosition,
                     for (int imode=1 ; imode < n_rz_azimuthal_modes ; imode++) {
                         // The factor 2 comes from the normalization of the modes
                         // The minus sign comes from the different convention with respect to Davidson et al.
-                        const Complex djt_cmplx = -2._rt * I*(i_new-1 + i + xmin*dxi)*wq*invdtdx/(amrex::Real)imode*
-                                                  (sx_new[i]*sz_new[k]*(xy_new - xy_mid) + sx_old[i]*sz_old[k]*(xy_mid - xy_old));
+                        const Real djt_coeff = -2._rt*(i_new-1 + i + xmin*dxi)*wq*invdtdx/(amrex::Real)imode;
+                        // For single precision, s_new and s_old are made single since multiplication of a double
+                        // and a single complex (the xy_ varianles) has not been implemented.
```

Comment:
```suggestion
                        // and a single complex (the xy_ variables) has not been implemented.
```

Reply:
Confusing though, it's defined as Complex of real, so includes SP:
https://github.com/ECP-WarpX/WarpX/blob/20.10/Source/Utils/WarpX_Complex.H#L21-L22